strapi/strapi
# chore(content-releases): add tracking

### What does it do?

Add tracking events to content-release events.

### How to test it?

1. Create a new release
2. You should see a new tracking request on the network tab

1. Publish a release
2. You should see a new tracking request on the network tab with the right counts
